moveit_core/robot_state/CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ if(CATKIN_ENABLE_TESTING)
   catkin_add_gtest(test_robot_state test/robot_state_test.cpp)
   target_link_libraries(test_robot_state ${catkin_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${MOVEIT_LIB_NAME})
 
+  add_executable(robot_state_benchmark test/robot_state_benchmark.cpp)
```

Comment:
> Except will this auto-run as a test in the CI? it doesn't make sense to me having useless tests run

@davetcoleman The binary is only declared as an executable, not as a test here.

Reply:
Ah ok. Can you add a note to the top of robot_state_benchmark.cpp simplying saying this is not a real test that gets auto-run by CI?